src/linear_algebra/basic.lean

```diff
@@ -2102,10 +2102,47 @@ end
 
 end field
 
+/-- Given `f : M₁ ≃ₗ[R] M₂` and `U` a submodule of `M₂`, `f.comap U` is the
+induced `linear_equiv` from `U.comap f.to_linear_map` to `U`. -/
+def comap [ring R] [add_comm_group M] [module R M]
```

Comment:
Does `@[simps]` work on this definition? As far as I can tell, you don't unfold the definition of `comap`, but if you do, could you add any of those results as a lemma?
```suggestion
@[simps]
def comap [ring R] [add_comm_group M] [module R M]
```

Reply:
`simps` works great and it generates the `apply` lemmas